controllers/managedocs_controller.go

```diff
@@ -1250,9 +1246,10 @@ func (r *ManagedOCSReconciler) reconcileK8SMetricsServiceMonitorAuthSecret() err
 		if err := r.own(r.k8sMetricsServiceMonitorAuthSecret); err != nil {
 			return err
 		}
-		var auth map[string][]byte = nil
-		var err error = nil
-		if auth, err = r.readGrafanaV1Secret(); err != nil {
+		// var auth map[string][]byte = nil
+		// var err error = nil
```

Comment:
if we are not using it why not just remove it?

Reply:
Can we resolve this?